.github/actions/terraform-deploy/action.yml

```diff
@@ -62,3 +62,12 @@ runs:
       run: |
         ./scripts/deploy.sh ${{ inputs.env-file }} <<< init
         ./scripts/deploy.sh ${{ inputs.env-file }} <<< deploy
+
+    - name: Output workflows API endpoint
+      id: output_workflows_endpoint
+      shell: bash
+      working-directory: ${{ inputs.dir }}
+      run: |
+        cd ./infrastructure
+        terraform output -json workflows_api > ${HOME}/terraform_outputs.json
+        cat ${HOME}/terraform_outputs.json
```

Comment:
Should this have an `output` defined? 

I know jobs explicitly define an output, not sure what actions do

Reply:
Great question! We can define an output like this [example does](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action#creating-an-action-metadata-file) in the docs, but it also isn't necessary since we can get the value from the `terraform_outputs.json`. Do you have a preference @ciaransweet ?

Reply:
I think it depends where you want those output values to be

I dont imagine workspace files are serialised across jobs to be accessible? But explicit outputs can be shared between jobs right?

Reply:
Yeah, `terraform_outputs.json` is available to be read within the github workflows that it is run in https://github.com/NASA-IMPACT/veda-deploy/actions/runs/10184498090/job/28173565851 so I think we can forgo adding an `output` and, instead, read from the file directly!